.github/workflows/python-publish.yml

```diff
@@ -6,6 +6,9 @@ name: Upload Python Package
 on:
   release:
     types: [created]
+  pull_request:
```

Comment:
This should be:
```on:
  release:
    types: [created]
```

Not this:
```
on:
  release:
    types: [created]
  pull_request:
    branches:
      - master
```

Reply:
I understand you may want to change this to test the github action is working on your branch before we merge this, of course!

Reply:
Yes I will remove this when we will merge this PR. Thanks for comment.